docs/clients/brushless_drive.rst

```diff
@@ -105,72 +105,116 @@ Message Table
+| 44     | emf                           | get            | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 45     | volts_at_max_amps             | get            | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 46     | slew_volts_per_second         | get, set, save | float     | V        |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 47     | slew_enable                   | get, set, save | float     | V        |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 48     | motoring_supply_current_limit | get, set, save | float     | fix16_t  |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 49     | regen_supply_current_limit    | get, set, save | float     | fix16_t  |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 50     | supply_current_limit_enable   | get, set, save | uint8     | bool     |                                                                                                                                      |      |
```

Comment:
Description: "Enables or disables dynamic supply current limiting. Limiting is enabled when true, and disabled when false."

No units, its a boolean, unless you consider that a type of unit

Reply:
we have 'bool' as a type of unit in other client entries